app/Models/User.php

```diff
@@ -112,4 +124,32 @@ public function hasAnyRole(): bool
     {
         return count($this->roles->pluck('name')->toArray()) > 0;
     }
+
+    public function getTotalRatingsAttribute()
+    {
+        $labels = array(
+            Rating::CREATIVE_LABEL,
+            Rating::FLEXIBLE_LABEL,
+            Rating::FRIENDLY_LABEL,
+            Rating::HELPFUL_LABEL,
+            Rating::PREPARED_LABEL
+        );
```

Comment:
This is the PHP 5.x array format, lets use the modern format instead: 
```suggestion
        $labels = [
            Rating::CREATIVE_LABEL,
            Rating::FLEXIBLE_LABEL,
            Rating::FRIENDLY_LABEL,
            Rating::HELPFUL_LABEL,
            Rating::PREPARED_LABEL
        ];
        
```